pkg/controlplane/authz/controllers.go

```diff
@@ -31,10 +31,24 @@ func CreateControllers(mgr *Manager, controllerManager ctrl.Manager, crdMode boo
 			Name:   "authz.access-policy",
 			Object: &v1alpha1.AccessPolicy{},
 			AddHandler: func(ctx context.Context, object any) error {
-				return mgr.addAccessPolicy(object.(*v1alpha1.AccessPolicy))
+				return mgr.AddAccessPolicy(object.(*v1alpha1.AccessPolicy), false)
 			},
 			DeleteHandler: func(ctx context.Context, name types.NamespacedName) error {
-				return mgr.deleteAccessPolicy(name)
+				return mgr.DeleteAccessPolicy(name)
```

Comment:
shouldn't this also have a bool flag to differentiate privileged and normal?
can a user specify a `metadata.namespace` for privileged access policies as well? 
The metadata field allows it, but would k8s API reject it as invalid or ignore the namespace (but retain it when Getting)?

I guess I'm confused by why it's needed in `AddAccessPolicy` but not in other methods.

Reply:
I put the flag back in place